components/BlurhashCanvas.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <canvas ref="canvas" :width="width" :height="height" />
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { decode } from 'blurhash';
+
+export default Vue.extend({
+  props: {
+    hash: {
+      type: String,
+      required: true
+    },
+    width: {
+      type: Number,
+      default: 32
```

Comment:
@MrTimscampi 'm late to the review party, but why 32 and not the ones used in current web? Any way to use workers is already sorted out? Because otherwise same perf problems that applied in web will happen here.

Reply:
It's doable, but it's currently fast enough to not be a noticeable issue. Even on a full library of 500 items, the query to the server takes longer than the blurhash, which is pretty much instant with Vue. I think we can start looking into this when the rest of the client is faster in regards to loading stuff (Same for unloading images).

As for the 32, it's the default mentioned on Blurhash's page. It's customizable all the way down to where the image is displayed. Same thing as with the worker, I prefer to leave it as a default while there are perf bottlenecks elsewhere, then go back to optimize once we're trying to make it as fast as we can :smiley: